engine/apps/public_api/serializers/escalation_policies.py

```diff
@@ -143,18 +151,10 @@ def to_representation(self, instance):
         result = super().to_representation(instance)
         result = self._get_field_to_represent(step, result)
         if "duration" in result and result["duration"] is not None:
-            result["duration"] = result["duration"].seconds
+            result["duration"] = int(float(result["duration"]))
```

Comment:
Is this conversion needed? (from what I see in the field definition, the `to_representation` should return the right repr?)

Reply:
Without this the DurationSecondsField returns a float as a string:
```
{
      "position": 0,
      "id": "E6J46FYHBXGUW",
      "escalation_chain_id": "FDWP4BEVEZD9C",
      "type": "wait",
      "duration": "111.0"
    },
```

Reply:
Probably don't really need to use that DurationSecondsField but I used it to match the other internal API serializer.